packages/paste-core/components/alert/__tests__/index.spec.tsx

```diff
@@ -66,12 +65,12 @@ describe('Alert', () => {
   describe('Variant neutral', () => {
     it('Should have no accessibility violations', async () => {
       const {container} = render(
-        <Theme.Provider theme="console">
+        <CustomizationProvider baseTheme="default">
```

Comment:
Close but not quite there. We should use the global Mock theme which is a static fixture. https://github.com/twilio-labs/paste/blob/main/packages/paste-core/components/menu/__tests__/customization.spec.tsx#L15

Reply:
Updated.